paasta_tools/paasta_cli/schemas/chronos_schema.json

```diff
@@ -0,0 +1,11 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "http://paasta.readthedocs.org/en/latest/yelpsoa_configs.html#chronos-clustername-yaml",
+    "type": "object",
+    "minProperties": 1,
+    "additionalProperties": {
+        "type": "object",
+        "properties": {},
```

Comment:
Yea yea, do it on a different pass. Here I did kind of the bare minimum with the json schema because I knew it was in flux. It has stabilized by now though.